src/js/utils/ui.js

```diff
@@ -250,6 +244,13 @@ const eventRegisters = {
         initInteractionListeners(ui);
     },
     tap(ui) {
+        if (OS.iOS && OS.version.major < 11) {
```

Comment:
thoughts on a file scoped var to gate whether or not this was added so we only have to do it once? might be overkill given the below is prob fast but could save an ms or two on every tap after the first